src/custom_node_library_internal_manager_wrapper.cpp

```diff
@@ -0,0 +1,27 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "custom_node_library_internal_manager_wrapper.hpp"
+
+namespace ovms {
+
+void* getCLIMWrapperPtr(const std::shared_ptr<CNLIMWrapper>& wrapper) {
+    if (wrapper == nullptr) {
```

Comment:
We can simplify this to `wrapper ? wrapper->ptr : nullptr`.
If so, maybe we could use this line directly in all those places and get rid of `getCLIMWrapperPtr` entirely? 😋